src/fstapi.c

```diff
@@ -2536,6 +2536,18 @@ int fstWriterGetFseekFailed(void *ctx)
     return (0);
 }
 
+static int fstWriterGetFlushContextPendingInternal(void *ctx)
+{
+    struct fstWriterContext *xc = (struct fstWriterContext *)ctx;
+    return (xc->vchg_siz >= xc->fst_break_size) || (xc->flush_context_pending);
+}
```

Comment:
Internal functions can use the struct type directly to make them a bit safer to use. In the future this will also be applied to the external API (see #2).

```suggestion
static int fstWriterGetFlushContextPendingInternal(struct fstWriterContext *xc)
{
    return (xc->vchg_siz >= xc->fst_break_size) || (xc->flush_context_pending);
}
```